_rules/css-orientation-lock-media-queries-b33eff.md

```diff
@@ -0,0 +1,198 @@
+---
+id: b33eff
+name: Page has no orientation lock specified using CSS media queries
+rule_type: atomic
+description: |
+  This rule checks that page content is not locked to any specific display orientation using CSS media queries, and the content is operable in all display orientations.
+accessibility_requirements:
+ wcag20:1.3.4: # Orientation
+  forConformance: true
+  failed: not satisfied
+  passed: further testing needed
+  inapplicable: further testing needed
+input_aspects:
+- DOM Tree
+- CSS Styling
+authors:
+- Jey Nandakumar
+- Audrey Maniez
+---
+
+## Applicability
+
+The rule applies to any `element` that is [visible](#visible) and has CSS media query styles targeting the display orientation.
```

Comment:
The "visible" link does not appear to be working.

Reply:
I ensured that there is a visible glossary/ definition & that this works.